samples/csharp_dotnetcore/08.suggested-actions/SuggestedActionsBot.cs

```diff
@@ -145,7 +145,9 @@ private static string ProcessInput(string text)
         /// Creates and sends an activity with suggested actions to the user. When the user
         /// clicks one of the buttons the text value from the <see cref="CardAction"/> will be
         /// displayed in the channel just as if the user entered the text. There are multiple
-        /// <see cref="ActionTypes"/> that may be used for different situations.
+        /// <see cref="ActionTypes"/> that may be used for different situations. For this sample we have only used two.
+        /// Note that the button with MessageBack ActionType initializes Text instead of Value.
```

Comment:
If we introduce 2 Action types in the sample, we should explain the user the practical differences between both. 

Here is the right place for that. Why do we have multiple action types? If Message is the recommendation should the sample onlyuse message? Otherwise we need to explain when to use one or the other.

Fnally, the technical details such as 'Note that the button with MessageBack ActionType initializes Text instead of Value. However, Value can be initialized as an additional property.' Should probably go in line 161 at the code level/.